ci/travis-after-success.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -xe
 
+# Submit code coverage report
+curl -s https://codecov.io/bash | bash
```

Comment:
They could steal credentials to push to ocean D2 (that, thanks to GitHub great permissions granularity, right now it means having access to ALL our code).